docs/index.html

```diff
@@ -438,7 +438,7 @@ <h2 class="heading settled" data-level="1" id="intro"><span class="secno">1. </s
    <p>This specification provides a standardized way of creating <a data-link-type="dfn" href="#credential-schema" id="ref-for-credential-schema①">Credential Schemas</a> to be used in credentialing platforms, how to version them, and how to read them. Credential Schemas may apply to any portion of a Verifiable Credential. Multiple JSON Schemas may back a single Verifiable Credential, e.g. a schema for the <code>credentialSubject</code> and another for other credential properties.</p>
```

Comment:
Also in this line, I'm wondering about `id="ref-for-credential-schema①"` — is that `①` intensional?
```suggestion
   <p>This specification provides a standardized way of creating <a data-link-type="dfn" href="#credential-schema" id="ref-for-credential-schema①">Credential Schemas</a> to be used in credentialing platforms, how to version them, and how to read them. Credential Schemas may apply to any portion of a Verifiable Credential. Multiple JSON Schemas may back a single Verifiable Credential, e.g., a schema for the <code>credentialSubject</code> and another for other credential properties.</p>
```

Reply:
My understanding is that the docs/index.html file is generated automatically by `bikeshed`. My guess is that the numbering is intentional, since I see other elements with the same `id` property name.

Reply:
its true, its possible this is setup awkwardly.